app/modules/users/models.py

```diff
@@ -618,19 +619,29 @@ def get_asset_groups(self):
 
     @module_required('asset_groups', resolve='warn', default=[])
     def unprocessed_asset_groups(self):
+        from app.modules.asset_groups.models import (
+            AssetGroup,
+            AssetGroupSighting,
+            AssetGroupSightingStage,
+        )
+
```

Comment:
would love thoughts from @hwindsor on this swap-out in particular.  i _think_ this will get us a list of any (user-owned) AssetGroup which has one (or more) AssetGroupSighting that _is not stage=`processed`_.  but is this what this **really** is meant to return?  the previous code (specifically definition of `asset_group.is_processed()` and that being negated, may have a subtle difference i am missing here.

Reply:
What you describe is certainly the intention. If an asset group has any AGS that are not processed, the AG is not processed. 
We require a list of those to show the User what they need to sort out on the frontend